src/pages/how-to/netflow.mdx

```diff
@@ -0,0 +1,248 @@
+# Introduction
+
+NetBird’s **Traffic Events** feature provides a high-level view of traffic flows between network peers and resources.
+It captures connection events on the client (peer) side – for example, when `Peer A` connects to `Peer B` –
+allowing administrators to observe how devices communicate across the NetBird network​.
+The primary purpose of flow logging is to help network admins monitor network activity,
+detect unusual or unauthorized connections, and troubleshoot connectivity issues in their NetBird mesh VPN​.
+Unlike packet capture, **Traffic Events** record metadata about the traffic (addresses, ports, timestamps, etc.) rather than the contents,
+preserving privacy while still giving useful insight​.
+
+By focusing on client-side events, NetBird’s **Traffic Events** show what each peer is doing on the network.
+This includes which peers or services it is contacting, over which protocols, and when.
+Traffic Events are especially useful for verifying that access control policies are working as expected (e.g. confirming that a peer could reach an allowed resource,
+or that blocked traffic wasn’t forwarded).
+In essence, they provide an audit trail of network connections in your NetBird environment,
+helping administrators ensure the network is being used according to policy and to quickly identify
+any anomalies or needed configuration changes.
+
+# Data Collected on Peers
+
+When enabled, a NetBird peer will record metadata for each network flow that it participates in. The data collected typically includes:
+
+* **Timestamp/Duration**: When the flow started and ended (or how long it lasted).
+* **Source and Destination IP Addresses**: The IP of the peer (source) and the IP of the remote endpoint (destination). For peer-to-peer traffic, these will be the NetBird network IPs (e.g. 100.x.x.x addresses of each peer). For traffic to an external resource (like a private server or subnet), the destination might be an IP in that remote network​.
+* **Source and Destination Ports**: The network ports used by the connection (for TCP/UDP flows). ICMP flows will be identified by ICMP type rather than ports​.
+* **Protocol**: The protocol of the traffic, such as TCP, UDP, or ICMP​.
+* **Volume of Data**: The amount of data transferred, often measured in number of packets and bytes sent/received for the duration of the flow​.(This is sometimes referred to as packet/byte counting.)
+**[CHECK] Add all fields**
+
+Crucially, only `metadata` is logged, not the actual payload of the packets.
+NetBird does not record contents of communications – for example, it might log that a peer made an `HTTPS` connection
+to an IP, but it will `NOT` log any `usernames`, `passwords`, or data exchanged inside that connection.
```

Comment:
not sure if we need to explain this and use username and passwords

Reply:
removed